### ISSUE TITLE
About dialog credits are editable text

### ISSUE THREAD

**Issue by danhickok** [danhickok/timecards#1]
The credits textbox in the About dialog are editable.  This control should be locked, and tab order should default to the OK button instead of this textbox.

**Comment by danhickok** [comment]
Problem fixed in version 1.0.1.0.
